src/components/views/settings/tabs/user/HelpUserSettingsTab.tsx

```diff
@@ -69,17 +69,14 @@ export default class HelpUserSettingsTab extends React.Component<IProps, IState>
             });
     }
 
-    private getVersionInfo(): { appVersion: string; olmVersion: string } {
+    private getVersionInfo(): { appVersion: string; cryptoVersion: string } {
         const brand = SdkConfig.get().brand;
         const appVersion = this.state.appVersion || "unknown";
-        const olmVersionTuple = this.context.olmVersion;
-        const olmVersion = olmVersionTuple
-            ? `${olmVersionTuple[0]}.${olmVersionTuple[1]}.${olmVersionTuple[2]}`
-            : "<not-enabled>";
+        const cryptoVersion = this.context.getCrypto()?.getVersion() ?? "<not-enabled>";
 
         return {
             appVersion: `${_t("setting|help_about|brand_version", { brand })} ${appVersion}`,
-            olmVersion: `${_t("setting|help_about|olm_version")} ${olmVersion}`,
+            cryptoVersion: `${_t("setting|help_about|crypto_version")} ${cryptoVersion}`,
```

Comment:
I'm renaming the `Olm Version: x.x.x` field to:
- old crypto: `Crypto Version: Olm x.x.x`
- new crypto: `Crypto Version: Rust SDK xxx, Vodozemac xxx`